tests/unittests/test_sync_event_updates.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from parameterized import parameterized
+from unittest import mock
+import datetime
+from tap_stripe import Context, sync_event_updates
+
+MOCK_DATE_TIME = datetime.datetime.strptime("2021-01-01T08:30:50Z", "%Y-%m-%dT%H:%M:%SZ")
+MOCK_CURRENT_TIME = datetime.datetime.strptime("2022-04-01T08:30:50Z", "%Y-%m-%dT%H:%M:%SZ")
+
+class TestSyncEventUpdates(unittest.TestCase):
+    """
+    Verify bookmark logic and logger message of sync_event_updates.
+    """
+    @mock.patch('stripe.Event.list')
+    @mock.patch('singer.utils.now', side_effect = [MOCK_DATE_TIME, MOCK_DATE_TIME, MOCK_DATE_TIME])
```

Comment:
Can't we keep `return_value =  MOCK_DATE_TIME` instead of the side_effect list if it same every time?